lib/hint.rb

```diff
@@ -44,7 +44,9 @@ def which_machinery
     end
 
     def get_started(_options)
-      "You can get started by inspecting a system. Run:\n#{program_name} inspect HOSTNAME"
+      "You can get started by inspecting a system. Run:\n'#{program_name} inspect HOSTNAME'\n" \
+      "Specify user with sudo rights for connecting via SSH:\n" \
```

Comment:
I'd mention somehow that this is only necessary if you don't want to run it with root so that it's clear that you should never do `inspect --remote-user root ...`